### ISSUE TITLE
Is the kb/sec an average or current?

### ISSUE THREAD

**Issue by Neltulz** [DJATOM/x265-aMod#10]
I noticed that the kb/sec display goes up/down but then eventually levels out.  That leads me to believe that it is an average.

It would be nice to see what the current kb/sec is... and maybe have some kind of indicator....

for example:

``avg: 8650 kb/sec, cur: 9100 kb/sec``

Can this be done?

Also, could the same be done with FPS? example:

``avg: 17.80 fps, cur: 15.00 fps``

**Comment by DJATOM** [comment]
Its average, which is most meaningful. But I'm planning to add a switch for calculating current speed instead of of average (I believe not much prople needs it).

**Comment by Neltulz** [comment]
I think it would be useful if it showed both avg and current.

**Comment by 44vince44** [comment]
@Neltulz 🤣 agreed. the possibility is useful. However you could look at the ONLY OTHER OPEN thread here and relate to it / or support it. Because it's almost the same request...

**Comment by Neltulz** [comment]
> However you could look at the ONLY OTHER OPEN thread here and relate to it / or support it. Because it's EXACTLY the same request... Which makes this double post...

@44vince44 You're an idiot.

![image](https://user-images.githubusercontent.com/7741742/109198397-cc6b8c00-7763-11eb-8650-49f543702d5f.png)

If you look closely, you'll see that those titles are entirely different, and as such, are entirely different issues.

If you look at this:
![image](https://user-images.githubusercontent.com/7741742/109198481-e3aa7980-7763-11eb-811a-9cf27a172f7f.png)

You'll notice that is actually an edit, not the original issue.  Which means, that the creation of this issue as a separate issue from yours, entirely makes sense, since I added that information about average FPS after I had created the issue.

> Please, the first precaution before posting is to check that there is no other post discussing same thing... especially when there is one single open post only....

> READ before POSTING my friend...

I did read before posting, and decided that your issue was different from mine.

So kindly, go fuck yourself.

**Comment by 44vince44** [comment]
@Neltulz it was not necessary to insult me.

As for what you say, in the other post, there is mention of "Instant Statistics".

![SNAG-0000](https://user-images.githubusercontent.com/17828567/109201567-aa492c80-77aa-11eb-8d9f-6afd65126664.jpg)

Instant statistics, by common sense, includes instant fps.

I know you from StaxRip page where I've already mentioned that to you. You drop a request, then don't follow up, even ignore comments addressed to you, then a few days later you post again something without even looking at the comments on your previous post, in which your second request has already been addressed.

There is something I'm sure about: I'm a polite person, and certainly not an idiot.

**Comment by Neltulz** [comment]
> As for what you say, in the other post, there is mention of "Instant Statistics", and "kpbs".

I read the title of your issue, read the page, and decided it was different than what I was suggesting.  Especially since what I'm asking for is the display of both average and current kbps.  Also, my issue is a question, to which djatom answered.

> I know you from StaxRip page where I've already mentioned that to you. 

I'm aware of that.

> You drop a request, then don't follow up

You're citing something that happened in december.  I was out of state for 2-3 weeks without access to a computer.

> even ignore comments addressed to you

I don't have to respond to every comment that is addressed to me.

> then a few days later you post again something without even looking at the comments on your previous post

If you recall, that was because Stax76 was attempting to push out a stable build and hadn't resolved some major problems.  I was doing some last second testing to try and help him ensure that his stable build was in fact stable.  Once I determined it wasn't stable, I addressed it in his github issue and that's when I started getting insulted for just trying to be helpful.  I do look at the comments on my previous posts, but when they reach the point where they're pages and pages worth of comments, it becomes difficult to keep up.  And the whole point of this paragraph, is that those issues were **NOT** addressed because clearly Stax76 was about to release a build of StaxRip that clearly wasn't stable and didn't address all of the things that were talked about in my previous post.  It's ancient history, and you are here bringing it up over absolutely nothing.  The tiniest little thing.  What the fuck is wrong with you man.

> Now here, you simply didn't read the other post, and the screenshot above proves is.

That's actually 100% false.  I did read your issue.  I decided it wasn't the same thing.  This is what you call a disagreement.

> There are rules in the forums and the issue trackers. Everyone is asked to respect them:

> Do not be rude

If you have an issue with me double posting, take it up with the github owner, or here's a crazy thought: keep your thoughts to yourself.  Don't harass me about it in my threads.  You were the one initially being rude to me, and I responded back to you in a manner that was fitting.

> Do not double post

It wasn't a double post.

> There is something I'm sure about: I'm a polite person, and certainly not an idiot.

That is something I'm sure about - You are not a polite person.  You're practically stalking me and won't leave me alone and continue to disagree with me despite our github issues having entirely different titles.

> and certainly not an idiot

Agree to disagree.

I'm going to ask you again.  Delete your posts above, and I will delete mine, and we can clean up this github issue and pretend like this never happened.
